### PR TITLE
Feature/random product display

### PR DESCRIPTION
**Pull request by Devynnem**
Files changed: Single card, added error component, App
Change Summary: Added random function to App on page load. Added Error handling to SingleCard Component
Additional Questions/Concerns: 
Up Next: Routing, displaying Single Product Info

### CODE REVIEW

**Review by brendel-r — COMMENTED** [review]
great job with the error handling.

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -9,38 +9,53 @@ import CardsContainer from '../CardsContainer/CardsContainer';
...
 
 export class App extends Component<Props, State> {
-  constructor(name: string) {
-    super(name)
+  constructor(props: Props) {
+    super(props)
     this.state = {
       allMakeUp: [],
-
+      randomBrand: brandArray[Math.floor(Math.random() * brandArray.length)],
```
Comment (brendel-r) [inline src/components/App/App.tsx:29]: mind blown. great dry code!

**Comment by tdmburr** [comment]
This looks great. We can proceed to getting our last components established (productInfo) touch up some CSS - and get some routing going soon. We may need to filter or find data for UX manipulation.

**Review by brendel-r — APPROVED** [review]

**Review by tdmburr — APPROVED** [review]
Approved, comment is available.

**Review by CapCinematic — APPROVED** [review]

**Pull request merged**